test/examples/ucp_hello_world.c

```diff
@@ -598,7 +643,15 @@ ucs_status_t parse_cmd(int argc, char * const argv[], char **server_name)
                     "of the server (required for client and should be ignored "
                     "for server)\n");
             fprintf(stderr, "  -p port Set alternative server port (default:13337)\n");
-            fprintf(stderr, "  -s size Set test string length (default:16)\n");
+            fprintf(stderr, "  -s size Set test string length (default:16)\n");       
+            fprintf(stderr, "  -m <mem type>  memory type of messages\n");
+            fprintf(stderr, "                 host - system memory (default)\n");
+            if (mem_type_allocators[UCS_MEMORY_TYPE_CUDA].malloc != NULL) {
+                fprintf(stderr, "                 cuda - NVIDIA GPU memory\n");
+            }
+            if (mem_type_allocators[UCS_MEMORY_TYPE_CUDA_MANAGED].malloc != NULL) {
+                fprintf(stderr, "                 cuda-managed - NVIDIA cuda managed/unified memory\n");
```

Comment:
Make it consistent? The previous option reads "NVIDIA GPU", here it reads "NVIDIA cuda".